DNA_RNA_classes_first_commit.py

```diff
@@ -0,0 +1,44 @@
+class Dna(str):
+    
+    
+    def __init__(self, dna_seq):  
+        self.dna_seq = dna_seq
+        codes = 'AaCcGgTtRrWwSsYyKkVvHhDdBbNn'
+        for i in self:            
+            if i not in codes:
+                raise Exception('String is a DNA sequence')
+            
+    def gc(self):
+        self = self.upper()
+        gc_percent = round(((self.count('G') + self.count('C'))*100)/len(self), 1)
+        gc_percent_str = str(gc_content) + '%'
+        return gc_percent_str
+    
+    def reverse_complement(self):

```

Comment:
Doesn't "reverse_complement" imply that the sequence should be returned reversed? In our homework file we were asked to do just "complement", but googling "reverse_complement" says it should be reversed:)